File: banking-ios copy/Shared/Model/Accounts/Balance.swift
//
//  Balance.swift
//  laboratory0 (iOS)
//
//  Created by Evgeniy Raev on 7.10.21.
//

import Foundation

struct Balance: Codable, Hashable {
    let current: Double
    let available: Double
    let opening: Double
    let blocked: Double
    let overdraft:Double
}

File: bnctl-banking-android/app/src/main/java/tl/bnctl/banking/data/accounts/model/Balance.kt
package tl.bnctl.banking.data.accounts.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Balance(
    val current: Double,
    val available: Double,
    val blocked: Double,
    val opening: Double,
    val overdraft: Double
) : Parcelable
